front/app/components/UI/MultipleSelect/index.tsx

```diff
@@ -91,8 +91,10 @@ export default class MultipleSelect extends React.PureComponent<Props, State> {
           options={options}
           onChange={this.handleOnChange}
           isDisabled={disabled}
-          styles={selectStyles}
+          styles={selectStyles()}
+          menuPosition="fixed"
```

Comment:
It opens the menu back downwards on mobile, so it doesn't hide under the header of the modal (see Notion ticket).

Reply:
(I didn't add it to the changelog because it's hard to explain and an edge case. So it will cause more confusion and it's not worth it.)